app/models/ability.rb

```diff
@@ -93,7 +93,7 @@ def initialize(user)
   end
 
   def signed_in?(user)
```

Comment:
Given that change - or eventually even before - the method name actually does not make that much sense, right?

How about just switching it to `signed_up_and_confirmed?`

Would make then intention clearer.

Reply:
I think that's a good point, @klappradla I'll take care of this. Thank you!